packages/xdl/src/TurtleApi.ts

```diff
@@ -0,0 +1,235 @@
+import merge from 'lodash/merge';
+import ExtendableError from 'es6-error';
+import QueryString from 'querystring';
+import axios, { AxiosRequestConfig } from 'axios';
+import idx from 'idx';
+import { JSONObject, JSONValue } from '@expo/json-file';
+
+import TurtleConfig from './TurtleConfig';
+import { FormData } from '@expo/xdl';
+
+// These aren't constants because some commands switch between staging and prod
+function _rootBaseUrl() {
+  return `${TurtleConfig.api.scheme}://${TurtleConfig.api.host}`;
+}
+
+function _apiBaseUrl() {
+  let rootBaseUrl = _rootBaseUrl();
+  if (TurtleConfig.api.port) {
+    rootBaseUrl += ':' + TurtleConfig.api.port;
+  }
+  return rootBaseUrl;
+}
+
+export class ApiV2Error extends ExtendableError {
```

Comment:
remove all apiv2 references and make sure that we actually need all those parts(remove all unnecessary code that might have appeared during code copypasting).